buku

```diff
@@ -31,6 +31,7 @@ import signal
 import logging
 import inspect
 import atexit
+import ruamel.yaml
```

Comment:
Can this be handled without a new dependency? For example, the way we did in markdown without using any markdown library?